lib/index.coffee

```diff
@@ -51,6 +53,29 @@ streamToString = (stream) ->
 		.on 'end', ->
 			resolve(Buffer.concat(chunks).toString())
 
+setupMetrics = (metrics) ->
+	metrics.describe('gc_space_reclaimed', 'Disk space the GC must free')
+	metrics.describe('gc_number_images_to_remove', 'Number of images to be removed at a given time')
+	metrics.describe('gc_duration_milliseconds', 'Milliseconds taken to free GC the reclaimed space')
```

Comment:
This should be a histogram, with buckets given by

```
metrics.client.exponentialBuckets(4, Math.SQRT2, 29).map(Math.round);
```

which gives `ms` buckets giving a good exponential spread from 4 ms up to ~ 65 s

```
[ 4,
  6,
  8,
  11,
  16,
  23,
  32,
  45,
  64,
  91,
  128,
  181,
  256,
  362,
  512,
  724,
  1024,
  1448,
  2048,
  2896,
  4096,
  5793,
  8192,
  11585,
  16384,
  23170,
  32768,
  46341,
  65536 ]
```

Reply:
:+1: